backbone/src/main/res/layout/rsb_activity_full_screen_image.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/fullScreenImageContainer"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/rsb_black">
+
+    <ImageButton
+        android:id="@+id/fullScreenImageClose"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
+        android:background="@null"
+        android:contentDescription="Close"
+        android:src="@drawable/rsb_ic_close"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="HardcodedText" />
+
+    <ImageView
+        android:id="@+id/fullScreenImage"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="10dp"
+        android:layout_marginEnd="10dp"
+        android:contentDescription="Image"
+        android:scaleType="centerInside"
+        app:layout_constraintBottom_toTopOf="@id/fullScreenImageEditStep"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
```

Comment:
Shouldn't this be _below_ `fullScreenImageClose` ? If the image is taller than the screen, this will pin to the top, sitting on top of the close button or am I wrong?